shared/chat/conversation/list-area/index.native.tsx

```diff
@@ -218,58 +217,61 @@ const ConversationList = React.memo(function ConversationList(p: {
   const centeredOrdinal = Container.useSelector(
     state => Constants.getMessageCenterOrdinal(state, conversationIDKey)?.ordinal ?? -1
   )
-  const messageOrdinals = Container.useSelector(state =>
+  const _messageOrdinals = Container.useSelector(state =>
     Constants.getMessageOrdinals(state, conversationIDKey)
   )
-  const listRef = React.useRef<Kb.NativeFlatList<ItemType> | null>(null)
+
+  const messageOrdinals = React.useMemo(() => {
```

Comment:
have to do this else it doesn't work strangely, i see complaints about this on their repo. we used to do this before so its likely fine